automation-tests/test/specs/orgCreationAndAuthentication.e2e.ts

```diff
@@ -0,0 +1,264 @@
+    projectFolderPath = path.join(tempFolderPath, tempProjectName);
+
+    // Clean up the temp folder, just in case there are stale files there.
+    if (fs.existsSync(projectFolderPath)) {
+      utilities.removeFolder(projectFolderPath);
+      await utilities.pause(1);
+    }
+
+    // Now create the folder.
+    if (!fs.existsSync(tempFolderPath)) {
+      await utilities.createFolder(tempFolderPath);
+      await utilities.pause(1);
+    }
+
+    await utilities.createFolder(projectFolderPath);
+    await utilities.pause(1);
+  });
+
+  step('Run SFDX: Create Project', async () => {
+    const workbench = await (await browser.getWorkbench()).wait();
+
+    prompt = await utilities.executeQuickPick(workbench, 'SFDX: Create Project');
+    // Selecting "SFDX: Create Project" causes the extension to be loaded, and this takes a while.
+    await utilities.pause(10);
+
+    // Select the "Standard" project type.
+    let quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).not.toBeUndefined();
+    expect(quickPicks.length).toBeGreaterThanOrEqual(1);
+    expect(quickPicks.length).toEqual(3);
+    expect(await quickPicks[0].getLabel()).toEqual('Standard');
+    expect(await quickPicks[1].getLabel()).toEqual('Empty');
+    expect(await quickPicks[2].getLabel()).toEqual('Analytics');
+    await prompt.selectQuickPick('Standard');
+    await utilities.pause(1);
+
+    // Enter "TempProject-OrgCreationAndAuth" for the project name.
+    await prompt.setText(tempProjectName);
+    await utilities.pause(1);
+
+    // Press Enter/Return.
+    await prompt.confirm();
+
+    // Set the location of the project.
+    const input = await prompt.input$;
+    await input.setValue(projectFolderPath);
+    await utilities.pause(1);
+
+    // Click the OK button.
+    await utilities.clickFilePathOkButton();
+
+    // Verify the project was created and was loaded.
+    const sidebar = workbench.getSideBar();
+    const content = sidebar.getContent();
+    const treeViewSection = await content.getSection(tempProjectName.toUpperCase());
+    expect(treeViewSection).not.toEqual(undefined);
+
+    const forceAppTreeItem = await treeViewSection.findItem('force-app') as DefaultTreeItem;
+    expect(forceAppTreeItem).not.toEqual(undefined);
+
+    await forceAppTreeItem.expand();
+
+    // Yep, we need to wait a long time here.
+    await utilities.pause(10);
+  });
+
+  step('Run SFDX: Authorize a Dev Hub', async () => {
+    // This is essentially the "SFDX: Authorize a Dev Hub" command, but using the CLI and an auth file instead of the UI.
+    const workbench = await (await browser.getWorkbench()).wait();
+    await utilities.pause(1);
+
+    const authFilePath = path.join(projectFolderPath, tempProjectName, 'authFile.json');
+    const terminalView = await utilities.executeCommand(workbench, `sfdx force:org:display -u ${EnvironmentSettings.getInstance().devHubAliasName} --verbose --json > ${authFilePath}`);
```

Comment:
If the new project is what's under test here, should we be using the UI instead of the CLI? I definitely understand using the faster method in scratchOrg.ts though.